src/dashboard/TasksList.tsx

```diff
@@ -42,20 +42,14 @@ export const TasksList = () => {
                         fontSize="medium"
                     />
                 </Box>
-                <Link
-                    underline="none"
-                    variant="h5"
-                    color="textSecondary"
-                    to="/contacts"
-                >
+                <Typography variant="h5" color="textSecondary">
                     Upcoming Tasks
-                </Link>
+                </Typography>
+                <AddTask display="icon" selectContact />
```

Comment:
**issue:** The button hover state is more an ellipsis that a circle:

![Capture d’écran du 2024-08-08 08-53-00](https://github.com/user-attachments/assets/596c7138-b4d3-4bbf-9464-443f0ebd3dd3)